client/my-sites/marketing/do-it-for-me/landing.tsx

```diff
@@ -1,82 +1,160 @@
-import { Button, Card } from '@automattic/components';
+import { Card } from '@automattic/components';
+import formatCurrency from '@automattic/format-currency';
 import styled from '@emotion/styled';
+import { useTranslate } from 'i18n-calypso';
 import page from 'page';
 import React from 'react';
 import { useSelector } from 'react-redux';
-import CardHeading from 'calypso/components/card-heading';
-import FormattedHeader from 'calypso/components/formatted-header';
+import IT from 'calypso/assets/images/difm/IT-1.png';
+import consult1 from 'calypso/assets/images/difm/consult-1.png';
+import consult2 from 'calypso/assets/images/difm/consult-2.png';
+import dietician from 'calypso/assets/images/difm/dietician-1.png';
+import musician from 'calypso/assets/images/difm/musician-1.png';
+import photography from 'calypso/assets/images/difm/photography-1.png';
+import phsych from 'calypso/assets/images/difm/phsych-1.png';
+import restaurant from 'calypso/assets/images/difm/restaurant-1.png';
+import treeremoval from 'calypso/assets/images/difm/tree-removal.png';
+import { FullWidthButton } from 'calypso/my-sites/marketplace/components';
+import { getCurrentUserCurrencyCode } from 'calypso/state/currency-code/selectors';
 import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
 
 const VerticalsGrid = styled.div`
 	display: grid;
 	grid-template-columns: repeat( 3, 1fr );
-	gap: 2em;
+	gap: 0.75em;
+	text-align: center;
 	@media ( max-width: 960px ) {
 		grid-template-columns: 1fr;
 	}
 `;
 
+const Vertical = styled.h1`
+	margin-bottom: 16px;
+	padding: 16px 20px;
+	margin: 0 auto 10px;
+	padding: 16px;
+`;
+
+const VerticalHeading = styled.h1`
+	font-size: 1.25rem;
+	text-align: center;
+	margin: 1em 0 1em 0;
+	height: 4.25em;
+	border-bottom: 1px solid var( --color-border-subtle );
+	box-sizing: border-box;
+`;
+
+const ImageContainer = styled.div`
+	height: 150px;
+	overflow: hidden;
+	margin: 10px 0px;
+	@media ( min-width: 960px ) {
+		height: auto;
+	}
+`;
+
+const ButtonContainer = styled.div`
+	width: 100%;
+	@media ( min-width: 480px ) {
+		margin: 0 15px;
+		width: 200px;
+	}
+`;
+
+const Paragraph = styled.p`
+	margin: 16px;
+	font-size: 1rem;
+	text-align: justify;
+`;
+
+const Heading = styled.h2`
+	font-size: 1.5rem;
+	margin: 15px 15px 0;
+	padding: 0;
+`;
+
 export default function DoItForMeLandingPage(): JSX.Element {
 	const selectedSiteSlug = useSelector( getSelectedSiteSlug );
+	const translate = useTranslate();
+	const currencyCode = useSelector( ( state ) => getCurrentUserCurrencyCode( state ) ) ?? 'USD';
+	const displayCost = formatCurrency( 150, currencyCode, { stripZeros: true } );
```

Comment:
We can get the actual product cost using the `getProductCost` selector, for example:
https://github.com/Automattic/wp-calypso/blob/1caca47e768a3df3c930a084442f4bae66358238/client/my-sites/marketing/ultimate-traffic-guide/index.js#L65